### DISCUSSION TITLE
select2 field name with dot

### DISCUSSION THREAD

**Discussion by soluzione-software** [Laravel-Backpack/community-forum#959] [Q&A (Help)] [answered]
I have a problem in a filter.
i have a field that is present in 2 table (the same name)
when i put this field in a filter the system give me an error
Integrity constraint violation: 1052 Column 'update_year_id' in where clause is ambiguous

i try to put the sql correlation in fields name

['name' => 'wines.update_year_id',
                    'type' => 'select2',
                    'label' => __('update_year'),
                    'relation' => 'update_year',
                    'relation_id' => 'id',
                    'attribute' => 'name',
                    'in_evidence' => true
                ]

but with this name the select2 didn't open the option 
![image](https://github.com/Laravel-Backpack/community-forum/assets/49692407/c36b388a-9bf3-4a84-a1a4-766d115830f3)

why?
how could i put correlation in filter field name?

thanks

**Comment by jcastroa87** [comment, marked as answer]
Hello @soluzione-software 

You can check how to handle custom search logic in our [docs](https://backpackforlaravel.com/docs/6.x/crud-columns#custom-search-logic-for-columns)

Per example
```
'searchLogic' => function ($query, $column, $searchTerm) {
    $query->where('wines.update_year_id', 'LIKE', '%'.$searchTerm.'%');
 },
```


Cheers.

**Comment by jcastroa87** [comment]
Hi @soluzione-software please let me know if that example works for you, thanks

**Reply by soluzione-software** [reply]
Thanks @jcastroa87 , yes i solve.

**Comment by jcastroa87** [comment]
Im glad you solve it, i will close the issue, but please feel free to re-open or create a new one if needed.

Cheers.
